.travis.yml

```diff
@@ -0,0 +1,63 @@
+sudo: false
+
+language: php
+
+php:
+    - 7.2
+    - 7.3
+    - nightly
+
+before_install:
+    - phpenv config-rm xdebug.ini || true
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=${SYMFONY_VERSION}; fi
+
+install:
+    - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
+
+script:
+    - vendor/bin/phpunit
+
+jobs:
+    include:
+
+        - stage: Test
+          env: SYMFONY_VERSION='4.0.*'
+          php: 7.2
+
+        - stage: Test
+          env: SYMFONY_VERSION='3.4.*'
+          php: 7.2
+
+        - stage: Test
+          env: DEPENDENCIES=low
+          php: 7.2
+          install:
+              - travis_retry composer update --classmap-authoritative --no-suggest  --prefer-dist --prefer-lowest --prefer-stable
+
+        - stage: Code Quality
+          env: CODING_STANDARDS
+          script:
+              - vendor/bin/phpcs -p
+
+        - stage: Code Quality
+          env: STATIC_ANALYSIS
```

Comment:
these environment variable likely differentiate jobs, but could they just be listed in `script` inside the same `Code Quality` stage?

Reply:
Just a hack to see something to separate them on https://travis-ci.com/libero/content-api-bundle/builds/88082454 (idea from https://github.com/doctrine/doctrine2/blob/91346c41e71c8b4c4184abd6e109059d0fcecab6/.travis.yml#L86).

Reply:
Same result I was looking for :+1: